lib/stoplight/light/runnable.rb

```diff
@@ -26,6 +27,20 @@ def run
         end
       end
 
+      def lock(color)
+        state = case color
+                when Color::RED then State::LOCKED_RED
+                when Color::GREEN then State::LOCKED_GREEN
+                else raise Error::IncorrectColor
+                end
+
+        safely { data_store.set_state(self, state) }
+      end
+
+      def unlock
+        safely { data_store.set_state(self, Stoplight::State::UNLOCKED) }
+      end
```

Comment:
On the other hand, we cannot return a color in this method 🤔

Reply:
I'll think about what to return then :)

Reply:
@bolshakov, because we can't return color in this method think that we should return Light here.